### PR TITLE
Milestone 1.0.1.0

### PR DESCRIPTION
**Pull request by shubham9649**

### CODE REVIEW

File: admin/html/views/assistants.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ public function pageBtns()
 
     private function assistantsTable()
     {
-        echo '<table class="buddybot-org-assistants-table table table-sm">';
```
Comment (metagauss) [inline admin/html/views/assistants.php:32]: @shubham9649 why did we removed the table-sm class?

File: admin/initialchecks.php [hunk 2]
```diff
@@ -87,6 +88,34 @@ private function vectorStoreCheck()
                     // Translators: %s is url to Vector Store settings page in admin area. This should not be changed.
                     sprintf(wp_kses_post('<strong>BuddyBot Notice:</strong> No vector store detected. A vector store is required for BuddyBot to function properly. Please create one by clicking <a href="%s">here</a>.', 'buddybot-ai-custom-ai-assistant-and-chat-agent'), esc_url(admin_url('admin.php?page=buddybot-vectorstore')))
                 );
+            }else{
```
Comment (metagauss) [inline admin/initialchecks.php:91]: Please be consistent. `} else {` instead of `}else{`

File: admin/requests/settings.php [hunk 3]
```diff
@@ -183,7 +183,7 @@ function createVectorStore(apiKey){
                 let storeData = vectorstoreData();
 
                 const data = {
-                    "action": "createVectorStore",
+                    "action": "create_Vector_Store",
```
Comment (metagauss) [inline admin/requests/settings.php:186]: @shubham9649 please use consistent casing.

**Review by metagauss — CHANGES_REQUESTED** [review]
Please see comments @shubham9649

File: admin/html/views/assistants.php (hunk 1 above)
Comment (shubham9649) [reply admin/html/views/assistants.php:32]: we are getting the mobile responsiveness issues then jawed sir remove this to fix it.

**Pull request merged**